.github/workflows/cli-release.yml

```diff
@@ -92,6 +92,29 @@ jobs:
           prerelease: ${{ inputs.prerelease }}
           node-auth-token: ${{ env.CLI_NPMJS_TOKEN }}
 
+      # --- MCP Registry Publish ---
+      - name: Install MCP Publisher
```

Comment:
is this specific to the Github MCP Registry?

Reply:
No this is for the "official" registry.modelcontextprotocol.io registry: 

https://github.com/modelcontextprotocol/registry/blob/main/docs/explanations/ecosystem-vision.md
https://github.com/modelcontextprotocol/registry/blob/main/docs/guides/publishing/publish-server.md

Reply:
Could we update the name to mention which registry this task will publish to? it'll allow us to quickly distinguish when we add other registries

Reply:
Theoretically, this is the "master" repository that others will pull from (I don't think that happens yet). But yeah, I will try to make that clearer.